labgrid/driver/networkusbstoragedriver.py

```diff
@@ -52,7 +52,7 @@ def on_deactivate(self):
     def write_image(self, filename=None, mode=Mode.DD):
         if not self.storage.path:
```

Comment:
@Emantor Is this correct? If so, we cannot use the non-existent path in the Exception message below. Or should this be `self.storage.avail`?

Reply:
It is correct. This will print `None is not available` in the error case. I think its best to remove the variable from the error and just return a generic `device not available`